bin/verify-exercises

```diff
@@ -1,45 +1,60 @@
 #!/usr/bin/env bash
 
-temp_dir_base=$(mktemp -d)
+set -euo pipefail
 
-run_test() {
-    slug=$(basename $1)
-    temp_dir=${temp_dir_base}/${slug}
-    mkdir -p ${temp_dir}
-    
-    cp -r "$1/." $temp_dir
+original_dir=$(pwd)
 
-    outpath=$temp_dir/src/$slug.lfe
-    if [ "$slug" = "binary" ]; then
-        outpath=$temp_dir/src/binary-string.lfe
+get_deprecated_exercises() {
+    if [ -f "config.json" ]; then
```

Comment:
```suggestion
    if [[ -f "config.json" ]]; then
```

`[[` is a builtin while `[` is a command. `[[` is faster and not prone to expansion bugs.